### ISSUE TITLE
missing GAC reference to System.IO.Compression

### ISSUE THREAD

**Issue by raphil** [ExcelDataReader/ExcelDataReader#315]
I am trying to add this Nugget and getting the following error.
Failed to add reference. The package 'ExcelDataReader' tried to add a framework reference to 'System.IO.Compression' which was not found in the GAC. This is possibly a bug in the package. Please contact the package owners for assistance.

My project is targeting 4.7.1

**Comment by andersnm** [comment]
Hi, are you using VS2017? Can you try and update to latest 15.5.6? After some looking around I didn't get much wiser, but wonder if it's some kind of tooling issue with 4.7.1.

(I couldn't repro in either VS2015 and VS2017)

**Comment by raphil** [comment]
Yes, that very well may be the issue.
I am having a similar issue with a different nugget package and am in
middle of upgrading to latest VS 2017 as I believe (am hopeful that) this
very well may be the culprit.

Thanks

**Comment by andersnm** [comment]
Hi @raphil - did upgrading VS work out for you?

**Comment by raphil** [comment]
Yes it did.

That got rid of my compile time issues with both ExcelDataReader (and some
other packaged Nuggets which were dependent on .Net Standard 2.0 which was
causing issues with other libraries not being found\referenced).

Thank you for the suggestion and for following up!!

BTW,
I love ExcelDataReader!!!  It is quick and efficient.  I was using a
regular Oledb driver for importing spreadsheets.  However, we experienced
issues when there was mixed data types in a files (string and numeric) and
some of the data elements were being imported as blank (or null).  (This is
a documented\known bug with Oledb driver).  We switched over to
ExcelDataReader, and that imported everything perfectly and extremely
efficiently!!!!
